### PR TITLE
Refs [TASK][27733] [trainer] View detail user

### PR DESCRIPTION
**Pull request by ghost**
https://edu-redmine.sun-asterisk.vn/issues/27733
Screenshot from 2020-09-04 14-55-41

### CODE REVIEW

File: app/views/partials/users/_info_user.html.erb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<div class="form-group row">
+  <span class="col-sm-2 col-form-label font-weight-bold">
+    Name
+  </span>
+  <div class="col-sm-10">
+    <p class="col-form-label">item.</p>
+  </div>
+</div>
+
```
Comment (manhnt-0960) [inline app/views/partials/users/_info_user.html.erb]: end line

File: app/views/partials/users/_info_user.html.erb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<div class="form-group row">
+  <span class="col-sm-2 col-form-label font-weight-bold">
+    Name
```
Comment (manhnt-0960) [inline app/views/partials/users/_info_user.html.erb]: i18n

File: app/views/trainers/users/show.html.erb [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+        <div class="col-sm-10">
+          <p class="col-form-label"><%= @user.department.name %></p>
+        </div>
+      </div>
+      <div class="form-group row">
+        <span class="col-sm-2 col-form-label font-weight-bold">
+          <%= t ".label_office" %>
+        </span>
+        <div class="col-sm-10">
+          <p class="col-form-label"><%= @user.office.name %></p>
```
Comment (manhnt-0960) [inline app/views/trainers/users/show.html.erb]: dùng delegate

**Comment by ghost** [comment]
ready

File: app/views/trainers/users/show.html.erb (hunk 3 above)
Comment (ghost) [reply app/views/trainers/users/show.html.erb]: Vâng ạ

**Comment by ghost** [comment]
ready

**Pull request merged**